docs/source/index.md

```diff
@@ -12,57 +12,19 @@ api
-workflows using Github actions. The template includes CI workflows for code checks,
-documentation building and releasing python packages to PyPI.
-
-- [LICENSE.md](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository) for adding a license to the project repository.
-By default, this is the [Apache-2.0 license](http://www.apache.org/licenses/). Please
-change according to your project!
-
-- [docs](https://pradyunsg.me/furo/) for adding project documentation. Typically
-projects should have API reference documentation, user guides and tutorials.
-
-- [CONTRIBUTING.md](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors) with basic guidelines on how others can
-contribute to the repository.
-
-- [CODE_OF_CONDUCT.md](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-code-of-conduct-to-your-project) with standards on how the community engages in
-a healthy and constructive manner.
-
-- [.gitignore](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files)
-with some standard file extensions to be ignored by git. Please add/modify as necessary.
-
-- [codecov.yml](https://docs.codecov.com/docs/codecov-yaml) for using codecov.io to
-generate code coverage information for your repository. You would need to add codecov.io
-app as an [integration to your repository](https://docs.codecov.com/docs/how-to-create-a-github-app-for-codecov-enterprise).
-
-
-If you are starting a new project, you can navigate to the [Use this template](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template) button
-on the top right corner of the [template repository home page](https://github.com/VectorInstitute/FLorist)
-which will allow you to bootstrap your project repo using this template.
-
-Please check out the user guide page for more detailed information on using the
-template features. For exisiting projects, the [user guide](user_guide.md)
-can be followed to migrate to following the template more closely.
+FLorist is a platform to launch Federated Learning (FL) training jobs. Its goal is to bridge the
+gap between state-of-the-art FL algorithm implementations and their applications by providing a
+system to easily kick off, orchestrate, manage, collect, and summarize the results of
+[FL4Health](https://github.com/VectorInstitute/FL4Health) training jobs.
+
+As Federated Learning has a client and a server side, FLorist also has client and server-side
+“modes” to orchestrate the training process on both sides. FLorist’s will have long-running
+processes on the clients, which will be waiting for instructions from the FLorist server to
+start training. Once it starts, FLorist's server will monitor those client processes as well as its
```

Comment:
nit: Once the FL Server is started, FLorist will monitor both server and client processes .. 

Maybe we avoid using FLorist server to not conflate it with FL Server. Assuming we still want to have that separation